assets/diagram-js.css

```
@@ -568,6 +569,29 @@ marker.djs-dragger tspan {
   color: inherit;
 }
 
+.djs-popup-header-group {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.djs-popup-header-group .entry {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.djs-popup-header-group:not(:last-child):after {
```

You could use the [next sibling selector](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators#next-sibling_combinator):

```suggestion
.djs-popup-header-group + .djs-popup-header-group:before {
```

---

You don't like my 3 pseudo selector construct? 😢

---

Adjusted ✅